task-5/index.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useEffect, useState } from 'react';
+
+function App() {
+
+    var [fuel, setFeul] = useState(0);
+    var [alertText, setAlertTxt] = useState('Processing...');
+
+    useEffect(() => {

```

Comment:
```typescript
useEffect(() => {
  (async () => {
    await fetch(....)
  })(); ...
```
Instead of the callback hell, let's use IIFE (Immediately Invoked Function Expression) to enter async/await flow.